src/Sulu/Bundle/PreviewBundle/Infrastructure/Symfony/DependencyInjection/SuluPreviewExtension.php

```diff
@@ -51,37 +51,17 @@ public function prepend(ContainerBuilder $container): void
         $configs = $container->getExtensionConfig($this->getAlias());
         $config = $this->processConfiguration(new Configuration(), $configs);
 
-        if ($config['cache']['type']) {
-            if (!$container->hasExtension('doctrine_cache')) {
-                throw new \RuntimeException(
-                    'Deprecated "sulu_preview.cache" configuration used, but DoctrineCacheBundle was not registered.' . \PHP_EOL .
-                    'Register the DoctrineCacheBundle or use the "sulu_preview.cache_adapter" configuration.'
-                );
-            }
-
-            $container->prependExtensionConfig('doctrine_cache',
```

Comment:
it is not longer able to use the DoctrineCacheBundle as it is only supported with symfony 4: https://github.com/doctrine/DoctrineCacheBundle/blob/master/composer.json